pvnet/models/base_model.py

```diff
@@ -335,6 +338,7 @@ def save_pretrained(
                 repo_id=repo_id,
                 repo_type="model",
                 folder_path=save_directory,
+                branch=branch,
```

Comment:
have you check that this works?

Reply:
I cant see that here - https://huggingface.co/docs/huggingface_hub/v0.29.3/en/package_reference/hf_api#huggingface_hub.HfApi.upload_file

Reply:
I think it needs to be `revision`